templates/profile.html

```diff
@@ -1,4 +1,28 @@
 {% extends "templates/participant.html" %}
+
+{% block head %}
+    <link rel="payment" type="text/html" title="Give weekly gift on Gratipay"
+        href="https://gratipay.com/{{ username }}/" />
+    <meta name="twitter:card" content="summary" />
+    <meta name="og:url" content="https://gratipay.com/{{ username }}/" />
```

Comment:
`username` is `None` except for the actual profile page.

![screen shot 2015-01-07 at 10 08 44 pm](https://cloud.githubusercontent.com/assets/134455/5657254/d5a27946-96b9-11e4-8af4-a91f482c2066.png)